apps/richie/vars/all/main.yml

```diff
@@ -58,5 +58,6 @@ richie_extra_dependencies:
 richie_activate_http_basic_auth: false
 
 # -- volumes
+# volume sizes with power of 2 suffices (Ki, Mi, Gi, Ti...) e.g 2Gi and 0 for no volume.
```

Comment:
@jmaupetit I'm having second thoughts, isn't it better to have separate variables:
```yml
- richie_should_activate_media_volume: true
- richie_should_activate_static_volume: true
```

Reply:
@jmaupetit Done and ready for further review.